### DISCUSSION TITLE
Quyen Vu - hpsOpenMM tutorial

### DISCUSSION THREAD

**Discussion by giscus** [qvv5013/qvv5013.github.io#3] [General]
# Quyen Vu - hpsOpenMM tutorial

https://qvv5013.github.io/posts/openMM/hpsOpenMM_tutorial.html

**Comment by vuqv** [comment]
In the current implementation, the `system` class is separated from `openmm.System` class.
For information about `openmm.System` or all other kinds of stuff in python-format, check the installation directory (`conda` env) because openMM is written in C++, code on GitHub for Python is not completed
